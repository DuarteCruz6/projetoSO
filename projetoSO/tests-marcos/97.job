WRITE [(b5,91)(n6,85)(w6,11)]
WRITE [(m9,57)(b8,28)]
BACKUP
WRITE [(c4,29)(u6,6)]
WRITE [(z3,99)(m8,19)(n7,23)(s6,44)(b0,45)]
READ [m8]
READ [c4]
WRITE [(k0,96)(o5,26)(g2,75)(k6,77)(o6,63)]
WRITE [(z9,78)(g8,9)]
WRITE [(t4,91)(q3,53)]
WRITE [(t7,36)(w7,15)(m0,93)]
WRITE [(x3,28)(k5,99)(v3,95)(d3,50)(o7,40)]
READ [b8]
WRITE [(m3,78)(a2,9)(m1,66)(x8,44)]
WRITE [(a4,4)(r6,82)(u9,72)]
READ [m8]
WRITE [(m7,27)(c9,100)(s9,22)(u2,61)(o0,13)]
READ [z3]
WRITE [(q8,23)(q8,96)(j0,78)(r7,78)]
WRITE [(w0,60)]
READ [u2]
WRITE [(c8,79)(o8,6)(t2,98)(w2,22)(r9,38)]
DELETE [x8]
READ [o7]
WRITE [(k1,89)(s8,28)(x6,64)]
READ [x3]
WRITE [(x8,83)]
WRITE [(d7,0)(a9,66)(d5,36)(q2,32)]
BACKUP
WRITE [(i9,80)(v4,90)(p8,13)(j3,34)(e3,51)]
WRITE [(g0,84)(r2,12)(p0,53)]
SHOW
BACKUP
BACKUP
READ [x3]
WRITE [(r8,78)(n1,34)]
READ [m0]
WRITE [(l5,93)(x4,74)(i6,40)(n2,54)]
READ [n1]
BACKUP
WRITE [(f6,74)]
WAIT 1000
WRITE [(d8,52)(j7,100)]
SHOW
READ [m8]
WRITE [(a1,39)]
READ [k1]
WRITE [(p9,22)(z8,92)(z8,73)(l1,98)(a7,12)]
READ [j3]
WRITE [(g9,45)]
READ [s8]
DELETE [c9,d5]
WRITE [(e0,28)]
READ [v4]
READ [w7]
WRITE [(e7,78)(l0,16)(c1,89)(t5,60)]
WRITE [(k2,70)(i5,74)(m2,45)(f7,30)]
READ [k1]
WRITE [(r1,97)(w5,6)]
WRITE [(t1,0)(u7,34)(a6,78)]
WRITE [(s3,66)(p1,75)]
WRITE [(w1,81)(y3,14)(f0,95)(z5,49)(s0,63)]
WRITE [(p7,11)(f5,38)]
DELETE [k5]
WRITE [(k7,52)(i8,37)(u5,38)(n0,91)]
DELETE [b5]
READ [n0]
DELETE [o6,s6]
WRITE [(o4,53)(s4,94)(p4,27)]
WRITE [(r3,44)(d1,79)]
DELETE [z5,u5,j0,x4]
WRITE [(m5,6)(u8,10)(z6,38)(s1,38)]
BACKUP
WRITE [(o2,81)]
WRITE [(z0,89)]
SHOW
DELETE [a6]
WRITE [(j9,9)]
WRITE [(v7,20)]
WRITE [(b3,57)(l6,99)(q7,28)(h8,73)(p3,59)]
READ [i5]
DELETE [z9,p1,p3,f5,b0]
BACKUP
DELETE [i9,j9]
