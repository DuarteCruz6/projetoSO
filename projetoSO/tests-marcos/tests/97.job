WRITE [(b6,93)(e6,86)(v8,49)(o3,63)]
WRITE [(e7,70)]
WRITE [(r8,12)]
WRITE [(i4,75)(z7,30)(u1,28)(p9,44)(q3,30)]
READ [v8]
READ [r8]
WRITE [(y2,98)(t2,73)(g6,12)(c4,28)]
READ [u1]
WRITE [(w3,3)(b5,3)]
DELETE [b5]
WAIT 3000
DELETE [p9,o3,e7]
WRITE [(u4,84)]
WRITE [(o7,91)(p9,16)]
READ [e6]
READ [u1]
WRITE [(o4,14)(f8,22)(k7,39)(k2,47)(h7,29)]
DELETE [t2,g6,u4]
WRITE [(s1,95)(d5,1)(w1,89)(n8,86)(j4,44)]
READ [u1]
WRITE [(a4,53)(c5,66)(k8,75)(e9,38)]
DELETE [h7,e9,k8,b6,n8]
WRITE [(d8,32)(u7,85)(e1,58)(c6,78)]
SHOW
READ [o7]
READ [e1]
BACKUP
SHOW
READ [o4]
WRITE [(g6,69)(t1,12)(i8,5)(v4,17)(j2,29)]
READ [a4]
WRITE [(x5,98)]
READ [i8]
WRITE [(i1,3)(d7,97)(t8,32)(u6,61)]
READ [u7]
DELETE [t1]
SHOW
WRITE [(z2,13)]
BACKUP
DELETE [t8,c4,g6,p9,d8]
DELETE [u7,e6,w3,d7]
WAIT 3000
WAIT 1000
WRITE [(s7,22)(m4,68)(e6,69)]
SHOW
DELETE [x5,j2,c5,d5]
WRITE [(e4,44)(g3,31)]
READ [u1]
WRITE [(d2,7)]
DELETE [o4,q3]
WRITE [(t8,59)(r4,33)(e9,35)(v6,75)]
WRITE [(k5,45)(b4,49)(y7,68)(a1,79)]
SHOW
WRITE [(a0,41)(o2,10)(f7,7)(g5,16)]
READ [e1]
WRITE [(h5,6)]
WRITE [(e5,40)(v3,16)(r6,44)(p0,91)]
WAIT 1000
DELETE [r8,k5,y7]
READ [h5]
READ [i1]
SHOW
WRITE [(q8,77)(l4,12)(o5,16)]
DELETE [f8,e5,u6,c6]
DELETE [a4]
BACKUP
WRITE [(t7,23)(w4,10)(w7,94)]
WRITE [(c4,3)(t4,77)(q1,23)]
DELETE [j4]
WAIT 1000
READ [d2]
WRITE [(q0,81)(o4,89)]
WRITE [(y3,83)(o3,72)(w2,28)(c2,52)(n2,50)]
SHOW
READ [h5]
WRITE [(k0,95)(f8,32)]
