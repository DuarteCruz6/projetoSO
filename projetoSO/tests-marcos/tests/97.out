[(v8,49)]
[(r8,12)]
[(u1,28)]
[(e6,86)]
[(u1,28)]
[(u1,28)]
(a4, 53)
(c6, 78)
(c5, 66)
(c4, 28)
(d8, 32)
(d5, 1)
(e1, 58)
(e6, 86)
(f8, 22)
(i4, 75)
(j4, 44)
(k2, 47)
(k7, 39)
(o4, 14)
(o7, 91)
(p9, 16)
(q3, 30)
(r8, 12)
(s1, 95)
(u7, 85)
(u1, 28)
(v8, 49)
(w1, 89)
(w3, 3)
(y2, 98)
(z7, 30)
[(o7,91)]
[(e1,58)]
(a4, 53)
(c6, 78)
(c5, 66)
(c4, 28)
(d8, 32)
(d5, 1)
(e1, 58)
(e6, 86)
(f8, 22)
(i4, 75)
(j4, 44)
(k2, 47)
(k7, 39)
(o4, 14)
(o7, 91)
(p9, 16)
(q3, 30)
(r8, 12)
(s1, 95)
(u7, 85)
(u1, 28)
(v8, 49)
(w1, 89)
(w3, 3)
(y2, 98)
(z7, 30)
[(o4,14)]
[(a4,53)]
[(i8,5)]
[(u7,85)]
(a4, 53)
(c6, 78)
(c5, 66)
(c4, 28)
(d7, 97)
(d8, 32)
(d5, 1)
(e1, 58)
(e6, 86)
(f8, 22)
(g6, 69)
(i1, 3)
(i8, 5)
(i4, 75)
(j2, 29)
(j4, 44)
(k2, 47)
(k7, 39)
(o4, 14)
(o7, 91)
(p9, 16)
(q3, 30)
(r8, 12)
(s1, 95)
(t8, 32)
(u6, 61)
(u7, 85)
(u1, 28)
(v4, 17)
(v8, 49)
(w1, 89)
(w3, 3)
(x5, 98)
(y2, 98)
(z7, 30)
(a4, 53)
(c6, 78)
(c5, 66)
(d5, 1)
(e6, 69)
(e1, 58)
(f8, 22)
(i1, 3)
(i8, 5)
(i4, 75)
(j2, 29)
(j4, 44)
(k2, 47)
(k7, 39)
(m4, 68)
(o4, 14)
(o7, 91)
(q3, 30)
(r8, 12)
(s7, 22)
(s1, 95)
(u6, 61)
(u1, 28)
(v4, 17)
(v8, 49)
(w1, 89)
(x5, 98)
(y2, 98)
(z2, 13)
(z7, 30)
[(u1,28)]
(a1, 79)
(a4, 53)
(b4, 49)
(c6, 78)
(d2, 7)
(e9, 35)
(e4, 44)
(e6, 69)
(e1, 58)
(f8, 22)
(g3, 31)
(i1, 3)
(i8, 5)
(i4, 75)
(j4, 44)
(k5, 45)
(k2, 47)
(k7, 39)
(m4, 68)
(o7, 91)
(r4, 33)
(r8, 12)
(s7, 22)
(s1, 95)
(t8, 59)
(u6, 61)
(u1, 28)
(v6, 75)
(v4, 17)
(v8, 49)
(w1, 89)
(y7, 68)
(y2, 98)
(z2, 13)
(z7, 30)
[(e1,58)]
[(h5,6)]
[(i1,3)]
(a0, 41)
(a1, 79)
(a4, 53)
(b4, 49)
(c6, 78)
(d2, 7)
(e5, 40)
(e9, 35)
(e4, 44)
(e6, 69)
(e1, 58)
(f7, 7)
(f8, 22)
(g5, 16)
(g3, 31)
(h5, 6)
(i1, 3)
(i8, 5)
(i4, 75)
(j4, 44)
(k2, 47)
(k7, 39)
(m4, 68)
(o2, 10)
(o7, 91)
(p0, 91)
(r6, 44)
(r4, 33)
(s7, 22)
(s1, 95)
(t8, 59)
(u6, 61)
(u1, 28)
(v3, 16)
(v6, 75)
(v4, 17)
(v8, 49)
(w1, 89)
(y2, 98)
(z2, 13)
(z7, 30)
[(d2,7)]
(a0, 41)
(a1, 79)
(b4, 49)
(c2, 52)
(c4, 3)
(d2, 7)
(e9, 35)
(e4, 44)
(e6, 69)
(e1, 58)
(f7, 7)
(g5, 16)
(g3, 31)
(h5, 6)
(i1, 3)
(i8, 5)
(i4, 75)
(k2, 47)
(k7, 39)
(l4, 12)
(m4, 68)
(n2, 50)
(o3, 72)
(o4, 89)
(o5, 16)
(o2, 10)
(o7, 91)
(p0, 91)
(q0, 81)
(q1, 23)
(q8, 77)
(r6, 44)
(r4, 33)
(s7, 22)
(s1, 95)
(t4, 77)
(t7, 23)
(t8, 59)
(u1, 28)
(v3, 16)
(v6, 75)
(v4, 17)
(v8, 49)
(w2, 28)
(w7, 94)
(w4, 10)
(w1, 89)
(y3, 83)
(y2, 98)
(z2, 13)
(z7, 30)
[(h5,6)]
